APMrover2/GCS_Mavlink.cpp

```diff
@@ -1448,31 +1444,28 @@ void GCS_MAVLINK_Rover::handleMessage(mavlink_message_t* msg)
 
 #if MOUNT == ENABLED
     //deprecated. Use MAV_CMD_DO_MOUNT_CONFIGURE
-    case MAVLINK_MSG_ID_MOUNT_CONFIGURE:
-		{
-			rover.camera_mount.configure_msg(msg);
-			break;
-		}
+    case MAVLINK_MSG_ID_MOUNT_CONFIGURE: {
+        rover.camera_mount.configure_msg(msg);
+        break;
+    }
 
     //deprecated. Use MAV_CMD_DO_MOUNT_CONTROL
-    case MAVLINK_MSG_ID_MOUNT_CONTROL:
-		{
-			rover.camera_mount.control_msg(msg);
-			break;
-		}
+    case MAVLINK_MSG_ID_MOUNT_CONTROL: {
+        rover.camera_mount.control_msg(msg);
+        break;
+    }
 #endif // MOUNT == ENABLED
 
     case MAVLINK_MSG_ID_RADIO:
-    case MAVLINK_MSG_ID_RADIO_STATUS:
-        {
-            handle_radio_status(msg, rover.DataFlash, rover.should_log(MASK_LOG_PM));
-            break;
-        }
+    case MAVLINK_MSG_ID_RADIO_STATUS: {
+        handle_radio_status(msg, rover.DataFlash, rover.should_log(MASK_LOG_PM));
+        break;
+    }
 
     case MAVLINK_MSG_ID_LOG_REQUEST_DATA:
     case MAVLINK_MSG_ID_LOG_ERASE:
         rover.in_log_download = true;
-        /* no break */
+    /* no break */
```

Comment:
This comment was in the right place, it is there to mean that the previous case has no brake on purpose